src/routes/profile-route/profile-horizontal-menu.jsx

```diff
@@ -9,7 +9,7 @@ export const ProfileHorizontalMenu = () => {
     {
       title: "Me goblin",
       src: sidebarCrown,
-      link: "/profile/me",
+      link: "/profile",
```

Comment:
Might look better if the horizontal menu items are centered?
![image](https://github.com/jahorwitz/slopopedia/assets/110426130/b65066de-a903-4145-a1ef-9ec34a171301)
The extra space at the end looks off. Should probably have the same margins on each side so it centers better on the screen.